src/trainer/JasmineGraphTrainingSchedular.cpp

```diff
@@ -11,27 +11,29 @@ See the License for the specific language governing permissions and
 limitations under the License.
  */
 
-
-
 #include <sstream>
 #include <iostream>
 #include "JasmineGraphTrainingSchedular.h"
 #include "../metadb/SQLiteDBInterface.h"
 #include "../util/Conts.h"
 #include "../frontend/JasmineGraphFrontEnd.h"
-
+#include "../performancedb/PerformanceSQLiteDBInterface.h"
+#include <bits/stdc++.h>
```

Comment:
Remove the bits/stdc++.h header since it has not been used

Reply:
Done

Reply:
Ok